content/enterprise_influxdb/v1/about-the-project/release-notes.md

```diff
@@ -9,7 +9,154 @@ menu:
     parent: About the project
 ---
 
-## 1.10 [2022-09-07]
+## v1.11.3 [2023/10/12]
+
+{{% note %}}
+#### InfluxDB Enterprise and FIPS-compliance
+
+**InfluxDB Enterprise 1.11+** introduces builds that are compliant with 
+[Federal Information Processing Standards (FIPS)](https://www.nist.gov/standardsgov/compliance-faqs-federal-information-processing-standards-fips)
+and adhere to a strict set of security standards. Both standard and FIPS-compliant
+InfluxDB Enterprise builds are available. For more information, see
+[FIPS-compliant InfluxDB Enterprise builds](/enterprise_influxdb/v1/introduction/installation/fips-compliant/).
+{{% /note %}}
+
+### Bug Fixes
+
+- Surround binary expressions in parenthesis.
+- Miscellaneous build-system and continuous integration fixes.
+
+### Other
+
+- Generate `DIGESTS.tag` for each release.
+- Update `golang.org/x/crypto` version to match `go.mod`.
+- Upgrade Go to 1.20.10.
+
+---
+
+## v1.11.2 [2023/09/28]
+
+### Features
+
+- Add crypto implementation information to `/debug/vars`.
+- Add tools for manual metadata changes.
+- Upload `influx_tools` binaries to `influxdata-unpublished`.
+- Explicitly run a FIPS POST and log results to ensure the POST is run.
+- Eliminate false FIPS audit failures for replaced modules.
+
+### Bug Fixes
+
+- Miscellaneous build system and continuous integration fixes.
+- Execute `compact-series-file` at startup.
```

Comment:
This may warrant more discussion.  @bnpfeife can explain where this is, and I can explain what it is.  Admins may not want this feature turned oned on, but that requires some script modifications.